pkg/bootstrap/platform/gcp.go

```diff
@@ -42,9 +42,11 @@ const (
 	GCEInstanceID        = "gcp_gce_instance_id"
 	GCEInstanceTemplate  = "gcp_gce_instance_template"
 	GCEInstanceCreatedBy = "gcp_gce_instance_created_by"
+	GCPQuotaProject      = "gcp_quota_project"
 )
 
 var GCPMetadata = env.RegisterStringVar("GCP_METADATA", "", "Pipe separted GCP metadata, schemed as PROJECT_ID|PROJECT_NUMBER|CLUSTER_NAME|CLUSTER_ZONE").Get()
+var gcpQuotaProjectVar = env.RegisterStringVar("GCP_QUOTA_PROJECT", "", "Allows specification of a quota project to be used in requests to GCP APIs.")
```

Comment:
does it make sense for GCPMetadata to allow an (optional) additional segment?

Reply:
```suggestion
var GcpQuotaProjectVar = env.RegisterStringVar("GCP_QUOTA_PROJECT", "", "Allows specification of a quota project to be used in requests to GCP APIs.").Get()
```

For consistency and to allow override from other packages

Reply:
OK. Updated to allow for override.